tests/text_utils_test.py

```diff
@@ -7,6 +7,7 @@ class WordCountTestCase(unittest.TestCase):
 
     def test_word_count_p_tags(self):
         self.assertEqual(2, text_utils.get_word_count('<p>foo<strong>s</strong></p><p>bar</p>'))
+        self.assertEqual(500, text_utils.get_word_count('<p>word</p>' * 500))
```

Comment:
Why not inline `<p>word</p><p>word</p><p>word</p>` ? Does it only break after word count reaches 500?

Reply:
It used to break after 40

Reply:
Can you add a comment explaining the "magic" breaking point of 40 in the code?